snap/hooks/install

```diff
@@ -123,7 +113,7 @@ if [ ! -f ${SNAP_DATA}/config/security-api-gateway/kong.conf ]; then
 fi
 
 # finally, disable services in the snap that aren't meant to come up by default
-# by default, we want the export-*, support-*, and device-virtual services off
-for svc in export-distro export-client support-notifications support-scheduler support-logging device-virtual device-modbus device-mqtt device-random; do
+# by default, we want the export-* and support-* off
```

Comment:
small nitpick, can we make this 
```
# by default, we want the export-*, support-*, and device-* services off
```

(since the device-mqtt, etc. are also disabled by default)